src/mjolnir/admin.cc

```diff
@@ -24,6 +24,7 @@ sqlite3* GetDBHandle(const std::string& database) {
       sqlite3_close(db_handle);
       return nullptr;
     }
+    // sqlite3_extended_result_codes(db_handle, 1);
```

Comment:
intentional?

Reply:
hm, let me uncomment. it will give better error codes/msgs